generator/extends.go

```diff
@@ -249,6 +251,50 @@ func (g *generator) parseExtendFunc(fn *types.Func, opts *ParseExtendOptions) er
 		ReturnError:      returnError,
 		ReturnTypeOrigin: fn.String(),
 	}
-	g.extend[xsig] = methodDef
+	if opts.IsMapExtend {
+		xsig.Id = fn.Name()
+		g.mapExtend[xsig] = methodDef
+	} else {
+		g.extend[xsig] = methodDef
+	}
+
+	return nil
+}
+
+// parseMapExtend prepares a list of mapExtend methods for use.
+func (g *generator) parseMapExtend(converterInterface types.Type, converterScope *types.Scope, methods []string) error {
+	for _, methodName := range methods {
+		parts := strings.SplitN(methodName, packageNameSep, 2)
+		var pkgPath, namePattern string
+		switch len(parts) {
+		case 0:
+			continue
+		case 1:
+			// name only, ignore empty inputs
+			namePattern = parts[0]
+			if namePattern == "" {
+				continue
+			}
+		}
+		// todo: handle other package path formats
+		pattern, err := regexp.Compile(namePattern)
```

Comment:
This shouldn't need a pattern, as it is not possible to define multiple methods that are used in mapExtend.

Reply:
remove method name pattern match in parse mapExtend process
 https://github.com/jmattheis/goverter/pull/25/commits/86abf4cb98b94958a1c3a14bc5b5a6fffe74d7cf